test/new-e2e/tests/installer/upgrade_scenario_test.go

```diff
@@ -76,7 +81,7 @@ const (
 	latestAgentImageVersion   = "7.54.1-1"
 
 	latestInstallerImageVersion = "7.55.0-installer-0.4.2-1"
-	latestInstallerVersion      = "7.55"
+	latestInstallerVersion      = "7.55.0-installer-0.4.2"
```

Comment:
Can't we get rid of these constants as well as the plane agentVersion since they are not used ?

Reply:
We don't use them anymore indeed